src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -103,7 +103,93 @@ Sentry.init({
 });
 ```
 
-Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry. Here's an example for your server component:
+Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry.
+
+### Using a Tunneling Server
+
+Here's an example for your server component:
+
+```js {tabTitle:Express}
+// Change this to suit your needs
+const SENTRY_HOST = "oXXXXXX.ingest.sentry.io";
+const SENTRY_KNOWN_PROJECTS = ["123456"];
+
+app.post("/bugs", async (req, res) => {
+  try {
+    const envelope = req.body;
+
+    const piece = envelope.toString().split("\n")[0];
+    const header = JSON.parse(piece);
+
+    const dsn = new URL(header.dsn);
+    if (dsn.hostname !== SENTRY_HOST) {
```

Comment:
m: I assume we do this so that the endpoint can't be abused. We should add a comment to describe this because strictly speaking `SENTRY_HOST` is not necessary.

Ditto for `SENTRY_KNOWN_PROJECTS`.

Reply:
hmm, this is based on the existing tunnel examples, where they have set this up in a similar way too 🤔 not sure if it helps to explain this more, or just bloats this up? IMHO we _can_ also just leave this away if we want to keep this as simple as possible, not sure how important this check is.

Reply:
No hard feelings either way. Let me downgrade this to logaf l. I think it makes sense for users to be aware of abuse potential. We can just leave it as is (without explanation) and add it if we see a lot of people asking about it, which is not gonna happen anyhow.

Removing it would probably be handing people a footgun which I don't particularly like.

Reply:
Okay nevermind let's keep this for sure!